src/Build.cmd

```diff
@@ -51,17 +72,20 @@ SET OutDir=%CMDHOME%\..\Binaries\%CONFIGURATION%
 
 set STEP=VSIX
 
-if "%BuildOrleansNuGet%" == "false" (
+if NOT "%BuildOrleansVsix%" == "true" (
```

Comment:
This is the most significant change that might require some tweaks to the CI build.